e2e/support/helpers/e2e-collection-helpers.ts

```diff
@@ -5,12 +5,43 @@ import {
   getFullName,
   popover,
 } from "e2e/support/helpers";
+import type { Collection, CollectionId } from "metabase-types/api";
+
+export const createCollection = ({
+  name,
+  description = null,
+  parent_id = null,
+  authority_level = null,
+}: {
+  name: string;
+  description?: string | null;
+  parent_id?: CollectionId | null;
```

Comment:
This is not strictly true.
`CollectionId` is one of the most misleading types we have. `parent_id` cannot be "personal", "users", or "trash". I'm almost 100% positive it cannot even be "root". So only `RegularCollectionId | null` seems to be correct here.

Reply:
5ed7ad7a8c3ae45dd4e90fc10e6abd34fbba4172